app/src/abi/ERC20.json

```diff
@@ -0,0 +1 @@
+../../../build/contracts/ERC20.json
```

Comment:
Let's just copy the ABI we need, similar to https://github.com/aragon/aragon-apps/blob/master/apps/finance/app/src/abi/token-balanceof.json

Older apps usually have placed separate function ABIs in separate files, but for simplicity we could just group the ones we need into a single file here.